tests/basics/bytes_count.py

```diff
@@ -52,3 +52,5 @@ def t():
     return True
 
 print(b"0000".count(b'0', t()))
+
+print(b'\xaa \xaa'.count(b'\xaa'))
```

Comment:
I know this is being very picky, but don't you think this new test belongs at the end of the block of tests above, on line 50?

Reply:
I made a new block (after line 50) with some more tests using start and length too.

Reply:
Looks good!